Packs/OctoxLabs/Integrations/OctoxLabs/OctoxLabs.py

```diff
@@ -0,0 +1,223 @@
+"""Base Integration for Cortex XSOAR (aka Demisto)
+
+This is an empty Integration with some basic structure according
+to the code conventions.
+
+Developer Documentation: https://xsoar.pan.dev/docs/welcome
+Code Conventions: https://xsoar.pan.dev/docs/integrations/code-conventions
+Linting: https://xsoar.pan.dev/docs/integrations/linting
+
+This is an empty structure file. Check an example at;
+https://github.com/demisto/content/blob/master/Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.py
+
+"""
+
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+from octoxlabs import OctoxLabs
+
+import requests
+from typing import Any, Dict, List, Callable, Optional
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+
+""" CONSTANTS """
+""" HELPER FUNCTIONS """
+
+
+def convert_to_json(obj: object, keys: List[str]) -> Dict[str, Any]:
+    return {k: getattr(obj, k, None) for k in keys}
+
+
+def run_command(octox: OctoxLabs, command_name: str, args: Dict[str, Any]) -> CommandResults:
```

Comment:
I advise you to look at other integrations to get reference to how we usually execute our commands (via the main function), for example, you can look at "hello_world.py".

Reply:
I reviewed other integrations but I think this is easy maintainable and testable.